apps/frontend/src/libs/components/slider/styles.module.css

```diff
@@ -0,0 +1,195 @@
+.container {
+	--gradient-physical: linear-gradient(180deg, #ffe307 0%, #fff8b5 100%);
```

Comment:
you can use css var() function here and get the color codes from variables.css

Reply:
This is useful for situation where PO will ask to change certain gradient color throughout the whole project

Reply:
Thank you for noticing, haven't thought of that